ja/mnist-for-beginners.md

```diff
@@ -0,0 +1,166 @@
+  <h3>層の構築</h3>
+  <p>ここでは各ハイパーパラメタ―の背景（活性化やweightInit）については取り上げませんが、それらの役割について簡単に触れておきましょう。ただし、これらがなぜ重要なのかを知りたい方は、<a href="http://jmlr.org/proceedings/papers/v9/glorot10a/glorot10a.pdf" target="_blank">Xavier Glorot and Yoshua Bengioによる論文</a>をお読みください。
+  <img src="/img/onelayer_labeled.png"><br>
+  <pre><code class="language-java">
+    .layer(0, new DenseLayer.Builder()
+            .nIn(numRows * numColumns) //入力データポイントの数
+            .nOut(1000) // 出力データポイントの数
+            .activation("relu") // 活性化関数
+            .weightInit(WeightInit.XAVIER) // 重みの初期化
+            .build())
+    .layer(1, new OutputLayer.Builder(LossFunction.NEGATIVELOGLIKELIHOOD)
+            .nIn(1000)
+            .nOut(outputNum)
+            .activation("softmax")
+            .weightInit(WeightInit.XAVIER)
+            .build())
+    .pretrain(false).backprop(true)
+    .build();
+  </code></pre>
+  <p>ところで隠れ層とは一体何なのでしょうか？</p>
+  <p>隠れ層の各ノード（図式中の円）はMNISTデータセットでは、手書きの数字の特徴を表しています。例えば、数字の「6」を見ているとしましょう。その数字を表す丸い角を表すノード、交差した曲線を表すノードなどが一つづあるのです。これらの特徴はモデルの係数の重要性によって重みを付与され、この手書きの数字が実際に「6」なのか推論するために各隠れ層で再度組み合わせられます。ノードの層が多ければ多いほど、より正確な推論に必要な複雑さやニュアンスを取り込むことができます。</p>
+  <p>「隠れ」層と言われるわけは、入力データがネット内に入り、決断が出てくるのを見ることができても、どのようにして、またなぜニューラルネットワークがデータを処理しているかが人間には解読不可能だからです。ニューラルネットワークのパラメーターは、機械のみが読み取ることのできる長い数字のベクトルに過ぎないのです。</p>
+</section>
+
+<section>
+  <h2 id="training">モデルのトレーニング</h2>
+  <p>モデルが構築できたら、トレーニング を開始しましょう。IntelliJの右上で、緑色の矢印をクリックします。この操作により、先に挙げたコードが実行されます。</p>
+  <img src="/img/mlp_mnist_single_layer_example_training.png"><br><br>
+  <p>これはハードウェアによっては、完了までに数分掛かることがあります。</p>
+</section>
+
+<section>
+  <h2 id="evaluating">結果を評価</h2>
+  <img src="/img/mlp_mnist_single_layer_example_results.png"><br><br>
+  <p>
+  <b>Accuracy（正答率）</b> - モデルが正しく識別したMNIST画像の割合<br>
+  <b>Precision（適合率）</b> - 真陽性の数を真陽性と偽陽性の数で割った値<br>
+  <b>Recall（再現率）</b> - 真陽性の数を真陽性の数と偽陰性の数で割った値<br>
+  <b>F1値</b> - <b>適合率</b>と<b>再現率</b>の加重平均<br>
+  </p>
+  <p><b>正答率</b>はモデル全体を測定します。</p>
+  <p><b>適合率、再現率、F1</b>はモデルの<b>適合性</b>を測定します。例えば、ある人がさらなる治療を求めないため、癌は再発しないだろうと（偽陰性）と推測するのは危険なことでしょう。このため、全体的に<b>正答率</b>が低めでも偽陰性（つまり適合率、再現率、F1が高め）を回避するモデルを選択するのが賢明でしょう。</p>
+</section>
+
+<section>
+  <h2 id="conclusion">最後に</h2>
+  <p>これで完了です！コンピュータビジョンが0ドメイン知識であるニューラルネットワークをトレーニングし、正答率の97.1%を達成しました。最先端のパフォーマンスは、これよりさらに優れており、ハイパーパラメターをさらに調整してモデルを改善させることができます。</p>
+  <p>Deeplearning4jは、その他のフレームワークと比較すると、以下の点で優れています。</p>
+  <ul>
+    <li>規模を広げて、Spark、Hadoop、Kafkaなどの主要なJVMフレームワークと統合させることができる。</li>
+    <li>分散CPU及び/または分散GPUで実行に最適化されている。</li>
+    <li>JavaやScalaのコミュニティに貢献している。</li>
+    <li>導入された企業様への商業用サポートを行っている。</li>
+  </ul>
+  <p>ご質問のある方は弊社のオンライン<a href="https://gitter.im/deeplearning4j/deeplearning4j" target="_blank">Gitter サポートチャットルーム</a>にてご連絡ください。</p>
+  <ul class="categorized-view view-col-3">
+    <li>
+      <h5>その他のDeeplearning4jのチュートリアル</h5>
+      <a href="http://deeplearning4j.org/neuralnet-overview">Introduction to Neural Networks</a>
```

Comment:
Some links are not in Japanese but all other links available in Japanese are updated from English ones.